common/constant/metadata.go

```diff
@@ -0,0 +1,7 @@
+package constant
+
+const (
+	MDMidazID       = "midaz_id"
```

Comment:
what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.